tests/handlers/test_agent.py

```diff
@@ -30,3 +41,45 @@ def test_post_agent_unauthenticated(test_client, db_session):
 
     assert response.status_code == 403
     assert response.json()['detail'] == "Permission denied."
+
+
+@pytest.mark.asyncio
+async def test_close_websocket_connect(
+        db_session, test_client, access_token, session_cookie,
```

Comment:
`test_client` requested but unused. As mentioned above you should be using it though.